qaframework-bdd-tests/cypress/integration/cucumber/step_definitions/oclsubscriptionmodule/ocl-subscription-module.js

```diff
@@ -60,13 +53,13 @@ Then('the user should copy the subscription url', () => {
 });
 
 When('the user visit the OCL module page', () => {
-  cy.visit('http://localhost:8080/openmrs/owa/openconceptlab/index.html#/subscription')
+  cy.visit('https://demo.openmrs.org/openmrs/owa/openconceptlab/index.html#/subscription')
```

Comment:
Any reason why you are using demo instead of local host.

Reply:
Any response on the above comment.

Reply:
Sure I reverted the server back to the local instance that is why it's showing `Outdated` label.